cicd/internal/op/maven.go

```diff
@@ -31,12 +27,6 @@ func RunMavenOnPom(pom string, cmd string, args ...string) error {
 	fullArgs = append(fullArgs, "-f", pom)
 	fullArgs = append(fullArgs, "-e")
 	fullArgs = append(fullArgs, args...)
-	modules := flags.ModulesToBuild()
```

Comment:
I understand modulesToBuild is currently used by workflows? e.g. https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/76e18cf58e288ecfec15b48e0f94f8fccbcd1250/.github/workflows/kafka-pr.yml#L141